conjure-java-core/src/main/java/com/palantir/conjure/java/services/dialogue/BlockingGenerator.java

```diff
@@ -74,6 +75,13 @@ private MethodSpec blockingClientImpl(EndpointDefinition def) {
                 .addModifiers(Modifier.PUBLIC)
                 .addParameters(params)
                 .addAnnotation(Override.class);
+
+        if (def.getDeprecated().isPresent()) {
+            methodBuilder.addAnnotation(AnnotationSpec.builder(SuppressWarnings.class)
+                    .addMember("value", "$S", "deprecated")
```

Comment:
I think this needs s/deprecated/deprecation

Reply:
Argh 🤦‍♂